src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx

```diff
@@ -42,86 +40,91 @@ export const CardTitle: React.FC<CardTitleProps> = ({
     chargedAmount,
     authorizedAmount,
   } = transaction;
-  const title = capitalize(transaction.name || "Transaction");
 
   return (
-    <DefaultCardTitle
-      className={className}
-      title={
-        <div className={classes.title}>
-          <TransactionLink href={transaction.externalUrl} className={classes.methodName}>
-            {transaction.externalUrl && (
-              <IconButton>
-                <LinkIcon />
-              </IconButton>
-            )}
-            {title}
-          </TransactionLink>
+    <Box width="100%" display="flex" justifyContent="space-between" alignItems="center">
+      {transaction.externalUrl ? (
+        <TransactionLink href={transaction.externalUrl} className={classes.methodName}>
```

Comment:
question: Did you consider refactor to new macaw/box and get rid of old styles

Reply:
Oops, missed it

Reply:
The button link component you've implemented works well here 😄